ldb/templatetags/ldb_tags.py

```diff
@@ -58,9 +67,23 @@ def __init__(self, fields, variable_name, form_variable):
 
     def render(self, context):
         form = template.Variable(self.form_variable).resolve(context)
-        new_form = copy.copy(form)        
+        new_form = copy.copy(form)
         new_form.fields = SortedDict([(key, value) for key, value in form.fields.items() if key in self.fields])
 
         context[self.variable_name] = new_form
 
         return u''
+
+
+@register.filter(name='has_group')
+def has_group(user, group_name):
+    try:
+        group = Group.objects.get(name=group_name)
+    except:
+        return False  # group doesn't exist, so for sure the user isn't part of the group
+
+    # for superuser or staff, always return True
+    if user.is_superuser or user.is_staff:
```

Comment:
Dit is niet waar in de enige usecase die we nu hebben. Ook al is iemand superuser (secretaris) die persoon hoeft alleen maar naar userman te kunnen als hij/zij ook pc.com is..